content/03.results.md

```diff
@@ -23,15 +23,15 @@ Along with the patient tumors, the Portal contains a small number of human tumor
 
 Each of the available samples contains summarized gene expression data from either single-cell or single-nuclei RNA sequencing.
 However, some samples also include additional data, such as quantified expression data from tagging cells with antibody-derived tags (ADT), such as CITE-seq antibodies [@doi:10.1038/nmeth.4380], or multiplexing samples with hashtag oligonucleotides (HTO) [@doi:10.1186/s13059-018-1603-1] prior to sequencing.
-Out of the over 700 samples, 95 have associated CITE-seq data, and 35 have associated multiplexing data.
+Out of the over 700 samples, 95 have associated CITE-seq data, and 35 have associated data with cell hashing.
 In some cases, multiple libraries from the same sample were collected for additional sequencing, either for bulk RNA-seq or spatial transcriptomics.
-Specifically, 182 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
+Specifically, 176 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
```

Comment:
```suggestion
Specifically, 182 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
```

Reply:
This was fixed when I merged in `main` in https://github.com/AlexsLemonade/ScPCA-manuscript/pull/173/commits/d746cae9992ee1b92629c415b0a56e249f5a9c5e